rest_framework-stubs/serializers.pyi

```diff
@@ -253,6 +253,9 @@ class ModelSerializer(Serializer, BaseSerializer[_MT]):
         self, kwargs: MutableMapping[str, Any], extra_kwargs: MutableMapping[str, Any]
     ) -> MutableMapping[str, Any]: ...
     def get_extra_kwargs(self) -> dict[str, Any]: ...
+    def get_unique_together_constraints(
+        self, model: _MT
+    ) -> Generator[tuple[set[tuple[str, ...]], Manager[_MT]], None, None]: ...
```

Comment:
```suggestion
    ) -> Iterator[tuple[set[tuple[str, ...]], Manager[_MT]], None, None]: ...
```

Reply:
@sobolevn Thank you very much for your feedback :) I have made the necessary revisions based on your input!